CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Change Log
 All notable changes to the "vscode-ansible" extension will be documented in this file.
 
+## 0.5.0
+#### New features
+- New command to generate sample for azure_rm_resource module. [PR 178](https://github.com/VSChina/vscode-ansible/pull/178)
```

Comment:
maybe to generate "samples", but not a big thing

Reply:
fixed